src/main/java/core/basesyntax/SaveData.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class SaveData {
```

Comment:
```suggestion
public class DataSaver {
```
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-verbs-for-classinterface-names)